real_estate/models/estate_property.py

```diff
@@ -0,0 +1,39 @@
+from odoo import fields, models
+import datetime
+
+class RealEstateProperty(models.Model):
+
+    _name = "estate_property"
```

Comment:
```suggestion
    _name = 'estate.property'
```
Nice I hope you followed the documentation that's nice thing for a newbie but as per the standard odoo convention, we follow a dot seperation when defining `_name` which orm translates `.` to `_`
reference - https://www.odoo.com/documentation/18.0/developer/reference/backend/orm.html#odoo.models.BaseModel._name

Also for the best practice it's better to use `""` for strings that will be displayed for the user and use `''` for technical terms/uses